src/arch/MovieTexture/MovieTexture_Generic.h

```diff
@@ -43,7 +52,7 @@ class MovieDecoder
 	/*
 	 * Get the currently-decoded frame.
 	 */
-	virtual void GetFrame( RageSurface *pOut ) = 0;
+	virtual bool GetFrame( RageSurface *pOut ) = 0;
```

Comment:
This might compile but not actually work, so please double check that we still want `= 0` here (I'm not sure either way if it should stay there or not).

Reply:
The = 0 forces the function definition to exist, otherwise it won't compile. This line is here to make sure the decoder defines `GetFrame`. The bool change here is intentional, signalling we got the last frame and should loop if desired. (That document is with FFMpeg's `GetFrame` function)